src/web/views/templates/page.njk

```diff
@@ -59,6 +60,19 @@
   </div>
 {% endblock %}
 
+{% block footer %}
+  {{ govukFooter({
+    meta: {
+      items: [
+        {
+          href: "/cookies",
+          text: "Cookies"
```

Comment:
If we are not doing translations in this PR then probably should leave a `TO DO` note to translate here as well

Reply:
Done.